.swiftformat

```diff
@@ -0,0 +1,42 @@
+# Stream rules
+--header "\n{file}\nCreated on {created}.\nCopyright © {year} Stream.io Inc. All rights reserved.\n"
```

Comment:
I'd suggest removing the `Created on` from the header. It's not important when the file was created.

Reply:
Don't you want that nostalgic feeling when looking at old files?
_Man, I remember creating this file... Has it been 4 years already?_

Reply:
no

Reply:
😅

Reply:
No good arguments from my side so I removed it